package.json

```diff
@@ -58,6 +58,7 @@
     "jest-environment-jsdom": "^28.1.0",
     "lerna": "^3.20.2",
     "openmrs": "latest",
+    "pinst": "^3.0.0",
```

Comment:
Do we need this dependency? My guess is no for this setup (since the monorepo package is not published.

Reply:
Thank you @ibacher for the catch, I removed it.